string location_name
---
geometry_msgs/PoseWithCovarianceStamped pose
bool success